### ISSUE TITLE
Azure AD B2C Password reset policy not working

### ISSUE THREAD

**Issue by alvaradojl** [Azure/azure-saas#224]
**Describe the bug**
A policy PasswordReset.xml has been installed on the Azure AD B2C instance but it doesn't seem to take effect.

**To Reproduce**
1. Go to any of the web applications that require signin/signup
2. Click on Forgot Password?
3. Nothing happens

**Expected behavior**
User should be redirected to an Azure AD B2C page where he/she can reset password.

**Comment by 1iveowl** [comment]
I believe you are right and that password reset has not yet been implemented.

**Comment by alvaradojl** [comment]
@1iveowl  actually I thought there was no policy, but in fact it is installed, maybe it's just not referenced from the base policy?
<img width="156" alt="Screenshot 2023-05-12 125752" src="https://github.com/Azure/azure-saas/assets/3758486/04ff6cff-1ae4-4a12-a148-939132db7a66">

**Comment by 1iveowl** [comment]
Correct, it's there, but not referenced in code. Thank you for the clarification.
